task1/reduceArr.js

```diff
@@ -0,0 +1,41 @@
+const notes = [
+  {
+    id: 1,
+    title: 'Recipe',
+    description: 'Ingredients include 2 eggs...',
+    pagesCount: 2,
+    isMarked: false,
+    access: []
+  },
+  {
+    id: 2,
+    title: 'Workouts',
+    description: '3 sets of squats...',
+    pagesCount: 1,
+    isMarked: true,
+    access: []
+  },
+  {
+    id: 3,
+    title: 'Passwords',
+    description: 'VISA ...',
+    pagesCount: 6,
+    isMarked: true,
+    access: []
+  },
+  {
+    id: 4,
+    title: 'To Do 2021',
+    description: '1. Learn JS...',
+    pagesCount: 3,
+    isMarked: false,
+    access: []
+  }
+]
+
+let initialValue = 0
+let sum = notes.reduce(function (accumulator, currentValue) {
```

Comment:
1. It will be hard to detect what is the sum for other engineers, maybe it's better to use `sumOfPages` or `sumOfNotesPages` instead? 
2. Why you defined `sum` as `let variable`? Where it's changed?
3. Why you didn't use arrow function here? 
4. What is `accumulator` and what is `currentValue`? 
5. I think there is no need to use a return statement inside the function body.

```suggestion
const sumOfPages = notes.reduce((sum, note) => sum + note.pagesCount, initialValue)
```